recipe/meta.yaml

```diff
@@ -14,7 +14,7 @@ source:
     - except.patch
 
 build:
-  number: 1001
+  number: 2
```

Comment:
build numbers should be increased until a version number is bumped.

Reply:
Ok, I didn't understand the historical about the 1000+ numbers and the small numbers.